testsuite/test-backup.sh

```diff
@@ -6,10 +6,17 @@ run restic backup "${BASE}/fake-data"
 run restic restore "$(basename "$RESTIC_REPOSITORY"/snapshots/*)" "${BASE}/fake-data-restore"
 dirdiff "${BASE}/fake-data" "${BASE}/fake-data-restore/fake-data"
 
-SNAPSHOT=$(run restic list snapshots)
+SNAPSHOT=$(restic list snapshots)
```

Comment:
So that `$SNAPSHOT` even contains an id when not in debug mode. Don't ask. :disappointed: